app/bundles/CoreBundle/Assets/json/regions.json

```diff
@@ -2178,11 +2178,53 @@
 		"Madaba"
 	],
 	"Kenya": [
-		"Nairobi Municipality",
-		"Coast",
-		"North-Eastern Kaskazini Mashariki",
-		"Rift Valley",
-		"Western Magharibi"
+		"Baringo County",
+		"Bomet County",
+		"Bungoma County",
+		"Busia County",
+		"Elgeyo-Marakwet County",
+		"Embu County",
+		"Garissa County",
+		"Homa Bay County",
+		"Isiolo County",
+		"Kajiado County",
+		"Kakamega County",
+		"Kericho County",
+		"Kiambu County",
+		"Kilifi County",
+		"Kirinyaga County",
+		"Kisii County",
+		"Kisumu County",
+		"Kitui County",
+		"Kwale County",
+		"Laikipia County",
+		"Lamu County",
+		"Machakos County",
+		"Makueni County",
+		"Mandera County",
+		"Marsabit County",
+		"Meru County",
+		"Migori County",
+		"Mombasa County",
+		"Murang'a County",
+		"Nairobi County",
```

Comment:
Should this perhaps be Nairobi City, per the list here: https://en.wikipedia.org/wiki/ISO_3166-2:KE?

Reply:
Seem to be used interchangeably. Happy to go with 'Nairobi City County' given official site uses this -> https://nairobi.go.ke/